README.md

```diff
@@ -129,11 +129,11 @@ emojis.
 
 The Stanford CoreNLP includes a way to retrain the neural network to recognize
 sentiment of sentence structures. You have to feed it a training set (their
-training set is ~8,000 sentences) and a development set that helps you tunne1
+training set is ~8,000 sentences) and a development set that helps you tune
 parameters of the neural net. Both sets have to be sentences that are manually
 turned into Penn Tree format.
 
-You can find FOSS Heartbeat's training set in [empathy/train.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [empathy/dev.txt](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).
+You can find FOSS Heartbeat's training set in [empathy/train.txt] and its development set in [empathy/dev.txt].
```

Comment:
Sorry I wasn't clear enough on the change here and on Line 200. If you would be willing to update that would be great, otherwise, I can merge and edit in a separate PR.

I did want to have the markdown link remain but since we are referencing files it would be helpful to put the file in backticks within the square brackets like so:

```
You can find FOSS Heartbeat's training set in [`empathy/train.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [`empathy/dev.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).
```

which should render like this:

You can find FOSS Heartbeat's training set in [`empathy/train.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/train.txt) and its development set in [`empathy/dev.txt`](https://github.com/sarahsharp/foss-heartbeat/blob/master/empathy-model/dev.txt).